src/adr/0040-generalize-event-bus.md

```diff
@@ -0,0 +1,30 @@
+#40. Generalize EventBus
+
+Date: 2020-06-11
+
+## Status
+
+Proposed
+
+## Context
+
```

Comment:
I had a very interesting discussion with Eugen Stan yesterday. 

Globally, we need to connect our architecture decisions with the Email business domain James is all about (and I tend to agree).

Here quota ratio search is rather abstract.

Maybe we can further explain what it is:

```
User Email storage usage is limited both in size and count via quotas (IMAP RFC-2087). In 
order to ease administating large user bases, the quota search extension allows administrator
to retrieve all users exceeding a given occupation ratio.

When searching for users by quota ratio if we set ...
```

I believe it helps connecting the decision with the domain and explains what it is all about.